inc/modules/export/prince/class-printpdf.php

```diff
@@ -85,9 +85,9 @@ protected function themeOptionsOverrides() {
 		$sass = \Pressbooks\Container::get( 'Sass' );
 
 		if ( $sass->isCurrentThemeCompatible( 2 ) ) {
-			$extra = "/* Print Overrides */\n\$prince-image-resolution: 300dpi; \n";
+			$extra = "/* Print Overrides */\n\$prince-image-resolution: auto,normal \n";
```

Comment:
We don't need to output this, do we? The default will be set in [assets/book/styles/variables/_media.scss](https://github.com/pressbooks/pressbooks/blob/05ac44820f1e582dafa89af9ab60521437252e30/assets/book/styles/variables/_media.scss#L43) and this doesn't actually change it.